Add docker gradle plugin defunct declaration (#573)
palantir/gradle-docker: readme.md

[![Build Status](https://circleci.com/gh/palantir/gradle-docker.svg?style=shield)](https://circleci.com/gh/palantir/gradle-docker)
[![Gradle Plugins Release](https://img.shields.io/github/release/palantir/gradle-docker.svg)](https://plugins.gradle.org/plugin/com.palantir.docker)

Disclaimer: This Repo is now Defunct
-------------

- This repo is on life support only - although we will keep it working, no new features are accepted;

> - It is now barely used internally at Palantir - Our internal gradle tooling should be used instead.

Reviewer: I'd remove the last sentence, as it's only applicable to internal folks!
```suggestion
- It is no longer used internally at Palantir.
```
